src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -517,6 +517,23 @@ public T WithTelemetry(ITelemetryConfig telemetryConfig)
             return (T)this;
         }
 
+        /// <summary>
+        /// Sets telemetry client for the application.
+        /// </summary>
+        /// <param name="telemetryClients">List of telemetry clients to add telemetry logs.</param>
+        /// <returns>The builder to chain the .With methods</returns>
+        public T WithTelemetryClient(params ITelemetryClient[] telemetryClients)
```

Comment:
>WithTelemetryClient [](http://example.com/codeflow?start=17&length=19)

SAL doesn't expect the client to be initialized before being passed in. I think you'd want to call initialize here for each client. It's idempotent so if it's already initialized it's a no-op.